calico_node/startup.py

```diff
@@ -266,7 +266,11 @@ def main():
         sys.exit(1)
 
     as_num = os.getenv("AS")
-    as_num = as_num or None
+    if os.getenv("USE_HOST_AS") == "true":
```

Comment:
@caseydavenport Yes that should work for my use case. However, won't your suggestion slightly change existing behavior? Currently, if calico is restarted with an empty ASN, `client.create_host` will explicitly delete the entry in etcd. With your suggestion, if there is an ASN already defined and the client is restarted with `AS=` then the existing value will be used.

On a fresh configuration, either approach will work for me, however. What do you guys suggest?